### ISSUE TITLE
Arc Control v1.68.4938.6 slows and then freezes after multiple rapid mouse button item selection clicks

### ISSUE THREAD

**Issue by BigNish7** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#320]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Intel Arc Control v v1.68.4938.6

### Processor / Processor Number [Required]

Intel Core i9 13900KS

### Graphic Card [Required]

Intel A770 LE

### GPU Driver Version [Required]

- [X] 31.0.101.4369
- [ ] 31.0.101.4335
- [ ] 31.0.101.4314
- [ ] 31.0.101.4311
- [ ] 31.0.101.4257
- [ ] 31.0.101.4255
- [ ] 31.0.101.4148
- [ ] 31.0.101.4146
- [ ] 31.0.101.4125
- [ ] 31.0.101.4123
- [ ] 31.0.101.4091
- [ ] 31.0.101.4090
- [ ] 31.0.101.4034
- [ ] 31.0.101.4032
- [ ] 31.0.101.3975

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[System Support Log - Sunday, 21 May 2023.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/11523690/System.Support.Log.-.Sunday.21.May.2023.txt)

### Description and steps to reproduce [Required]

I am having an issue with Intel Arc Control 1.68.4938.6.

I installed Arc Control along with the latest WHQL Arc driver (31.0.101.4369). It's the first time I have installed Arc Control for a while as it has had numerous issues for some time and I was happy to wait for it to be further developed and just installed the driver updates. However, I thought I would give it a go to see how much it had improved.

Unfortunately I am having an issue that after several rapid mouse clicks selecting different items, Arc Control slows dramatically in responding to the mouse clicks and then freezes, or at least seems to freeze as it takes longer than my patience waiting for it to respond. Note that just Arc Control is freezing - the rest of my desktop seems unaffected. 

When this occurs I hover my mouse over the Arc Control icon on the taskbar and right click on the Close Window option which kills the frozen Arc Control.

I installed the latest driver and Arc Control via the 'clean instal' option and to try to fix this issue, I uninstalled the driver using DDU and did a fresh install from scratch. This did not fix the issue.

I would have taken a video of this issue using the screen capture facility of Arc Control, but I am getting the 'Could Not Start Capture. Device operation failure. Make sure you're not connected to a VPN. Error code: -17". I have had this error before with an earlier version of Arc Control and then it was fixed with a later version, but now it seems to be broken again. I am not connected to a VPN.

The simplest way I have found to replicate the issue is to click on the 'Settings' gearwheel icon (most RH of the four icons in the LH top corner  of Arc Control) and then start clicking on, in no particular order, the 'App Preferences',  'Global Game Settings', 'System Info', 'Technical Support' and 'About' tiles. For me, within a short time, Arc Control's response to my mouse click commends slows rapidly and then freezes such that I have to kill Arc Control.

**Comment by Karen-Intel** [comment]
Hey @BigNish7 thank you for your submission!
I was able to reproduce the same behavior on a 13900k system + ARC A750 + Gfx Driver v.4369
I am reporting it ASAP.
Just FYI for the fix to be released might take a while (~3 to 6 months) but if we have news or updates sooner we will let you know in this post

Thanks a lot for your help!

Karen

**Comment by BigNish7** [comment]
Hi again @Karen-Intel 

I have been doing some further testing and think I have found what triggers this issue. It seems specific to Arc Control v1.68.4938.6.

In my further testing, I found that the slowing occurs/is triggered after I click the 'Hamburger' icon and, for example, turn the Studio feature OFF. Once I do this, I get the slowing and then Arc Control becomes so slow it effectively freezes and I have to kill it via closing it from the taskbar.

I also tested to see if I could get the slowing behaviour with the prior version of Arc Control (v1.66.4728.1) that came bundled with WHQL driver 31.0.101.4314. I am happy to say that I did not get the same slowing behaviour with this earlier version.

To show you in more detail what I am experiencing, I took two short videos. One showing the slowing after turning Studio OFF with 1.68.4938.6 and one showing no slowing after turning Studio OFF with 1.66.4728.1. In the 'slowing' video you will see that once the slowing has started it looks like I am just moving the mouse around over the tiles, but I can assure you I am clicking on the tiles, and other icons, but they are unresponsive.

In summary then, I have found for my PC:

1. Arc Control v1.66.4938.6 slows to the point of appearing frozen when using multiple rapid mouse clicks AFTER turning Studio OFF. I also rebooted and tested it turning Studio ON, and can confirm that this also triggers the slowing behaviour.

2. Arc Control v1.66.4728.1 does not show this slowing behaviour AFTER turning Studio OFF.

I sincerely trust that the above info is helpful.


https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/116610885/184b9a8e-85e7-4246-ac26-28a9886cfa4f


https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/116610885/217e7541-04a0-43dd-a427-42bff911bb23

**Comment by BigNish7** [comment]
Hi @Karen-Intel 

I recently installed the most recent BETA Driver/Arc Control combo (31.0.101.4575 and 1.70.5084.2) and am very happy to say that Arc Control v1.70.5084.2 does not have this issue re. slowing down after multiple mouse clicks. 

I am confident in saying that this behaviour is now resolved! 

Thank you very much for taking the time to independently ascertain the issue and for bringing about a successful resolution. 

Kindest regards,

BigNish7

**Comment by Karen-Intel** [comment]
Heey @BigNish7 it's a pleasure for us to be your voice internally. I'm glad it's working now!
I hope to hear from you soon, we love to collaborate with you guys and bring fixes little by little :)

@IGCIT can you help us close this one? Fix has been released and confirmed by customer in the latest beta (v.4575)

Karen
